mongodb/driver/manager/executewritebatch.xml

```diff
@@ -20,9 +20,11 @@
   </para>
   <para>
    A <classname>MongoDB\Driver\WriteBatch</classname> can be constructed with
-   multiple different types of write operations (e.g. updates, deletes, and
-   inserts) in the same batch. The driver will attempt to send as many operations
-   to the server as it can at the same time, optimizing roundtrips.
+   multiple write operations of varying types (e.g. updates, deletes, and
+   inserts). The driver will attempt to send similarly type operations to the
+   server at the same time to optimize round trips. For instance, a batch of
+   three inserts and one update might only require two round trips, since the
```

Comment:
Thats not really correct. It totally depends on the order in the batch - this could easily be 3 roundtrips, which is why I refrained from describing it in the next and had it in the example itself